module/components/Badge/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Badge as ReactstrapBadge } from 'reactstrap';
+
+const propTypes ={
+    /**
+     * Useful to extend the style applied to components.
+     */
+    classes: PropTypes.string,
+    /**
+     * The color of the component. It's using the theme palette when that makes sense.
+     */
+    color: PropTypes.string,
+    /**
+     * Adding the href prop will default the badge to a link
+     */
+    href: PropTypes.string,
+    /**
+     * If `true`, pill style will be added.
+     */
+    pill: PropTypes.bool
+};
+
+const defaultProps = {
+    pill: false,
+};
+
+export class Badge extends React.Component{
+
+
+    render(){
+
+        const { classes,
+                color,
+                pill,
+                href
+        } = this.props;
+
+        let materialClasses = [
+                                pill ? `badge-pill` : false
+                                ];
+
+        return(
+            <ReactstrapBadge className={ classNames(classes, materialClasses) } color={ color }  href={href}>
+                { this.props.children }
+            </ReactstrapBadge>
+        );
+    }
+
+}
+
+Badge.propTypes = propTypes;
+Badge.defaultProps = defaultProps;
+
+export default Badge;
```

Comment:
Please add a newline at the end for all files as per [eslint](https://eslint.org/docs/rules/eol-last) and [Airbnb](https://github.com/airbnb/javascript#whitespace--newline-at-end) spec